src/ig/IGExporter.ts

```diff
@@ -827,10 +876,52 @@ export class IGExporter {
             newResource.extension = configResource.extension;
           }
           this.ig.definition.resource.push(newResource);
+          this.artifacts.instances.push({
+            name: newResource.name,
+            url: instance.url,
+            description: newResource.description,
+            referenceKey
+          });
         }
       });
   }
 
+  private addResource(
```

Comment:
I might be missing something, but is there a reason you don't use this function in the `addPredefinedResouces` function as well? I know that function and `addResources` are very similar, so I was just curious if there to not.

Reply:
It looks like `addPredefinedResources` has some extra logic for things like overriding existing values, getting values from meta extensions, etc. So, I don't think I can make use of `addResource` in that function.